src/DataStructures.cpp

```diff
@@ -40,7 +40,9 @@ const parameter_info parameter_info_list[] = {
     {iCvmass,            "Cvmass",            "O", "J/kg/K",  "Mass specific constant volume specific heat",     false},
     {iCp0molar,          "Cp0molar",          "O", "J/mol/K", "Ideal gas molar specific constant pressure specific heat",false},
     {iCp0mass,           "Cp0mass",           "O", "J/kg/K",  "Ideal gas mass specific constant pressure specific heat",false},
-    {iSmolar_residual,   "Smolar_residual",   "O", "J/mol/K", "Residual molar entropy (sr/R = tau*dar_dtau-ar)",false},
```

Comment:
Please put this back, but maybe write sr/R = s(T,rho)-s^0(T,rho) instead of what I did

Reply:
will do